packages/discovery-react-components/src/components/StructuredQuery/components/RuleRow/RuleRow.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { ComboBox, TextInput } from 'carbon-components-react';
+import { Messages } from 'components/StructuredQuery/messages';
+import { structuredQueryRulesClass } from 'components/StructuredQuery/cssClasses';
+
+export interface RuleRowProps {
+  /**
+   * override default messages for the component by specifying custom and/or internationalized text strings
+   */
+  messages: Messages;
+}
+
+export const RuleRow: React.FunctionComponent<RuleRowProps> = ({ messages }) => {
+  const handleOnFieldChange = () => {
```

Comment:
probably can remove this stuff for this PR

Reply:
I added these in now since `onChange` is a required prop for `ComboBox` and I didn't want to see a warning in Storybook.